test/TestHelpers.cs

```diff
@@ -236,6 +241,46 @@ private static List<string> GetLogs_HelloWorldOrchestration_Inline(string instan
             return list;
         }
 
+        private static List<string> GetLogs_Activity_Gets_HttpManagementPayload(string instanceId, string[] functionNames, bool extendedSessions)
```

Comment:
Can we actually remove these log checks too? I think the other components of your end-to-end test are already sufficient.